acquisitions/src/main/resources/thunderjet/mod-orders/features/routing-list-print-template.feature

```diff
@@ -0,0 +1,161 @@
+  # for https://folio-org.atlassian.net/browse/MODORDERS-1026
+  Feature: Should check processing of printing routing list functionlity
+
+    Background:
+      * url baseUrl
+      * callonce loginAdmin testAdmin
+      * def okapitokenAdmin = okapitoken
+
+      * callonce loginRegularUser testUser
+      * def okapitokenUser = okapitoken
+
+      * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': 'application/json'  }
+      * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json'  }
+
+      * configure headers = headersUser
+
+      * callonce variables
+      * def routingListId = "a1d13648-347b-4ac9-8c2f-5bc47248b87e"
+      * def userId1 = '00bc2807-4d5b-4a27-a2b5-b7b1ba431cc4'
+      * def userId2 = '011dc219-6b7f-4d93-ae7f-f512ed651493'
+      * def templateId = '9465105a-e8a1-470c-9817-142d33bc4fcd'
+      * def orderId = callonce uuid1
+      * def poLineId = callonce uuid2
+      * def homeAddressTypeId = callonce uuid3
+      * def homeAddressLine1V1 = '1235 Iowa str'
+      * def homeAddressLine1V2 = '1550 Austin str'
+      * def officeAddressTypeId = callonce uuid4
+      * def officeAddressLine1V1 = '113 Law'
+      * def officeAddressLine1V2 = '143 Law'
+
+    Scenario: Create AddressType
```

Comment:
based on latest best practices better to use print to prepare data fro test instead of introducing new scenarious for it, for details please see reated discussion: https://github.com/folio-org/folio-integration-tests/pull/1309#discussion_r1546712372

Reply:
Thanks to notice, done